tests/factories/cht/generate.js

```diff
@@ -137,14 +168,29 @@ const createData = ({ healthCenter, user, nbrClinics, nbrPersons, useRealNames =
 const createHierarchy = ({ name, user = false, nbrClinics = 50, nbrPersons = 10, useRealNames = false }) => {
   const hierarchy = placeFactory.generateHierarchy();
   const healthCenter = hierarchy.get('health_center');
-  user = user && userFactory.build({ place: healthCenter._id, roles: ['chw'] });
+  const branch = hierarchy.get('district_hospital');
+  const branchContact = {
+    _id: 'fixture:user:user2',
+    name: 'Manager Ann',
+    phone: '+123456789'
+  };
+  const contact = {
+    _id: 'fixture:user:user1',
+    name: name,
+    phone: '+12068881234'
+  };
```

Comment:
Like this, it stays closer to the definition, so when reading code, it will help imaging how data is related from the start 
```suggestion
  const branch = hierarchy.get('district_hospital');
  branch.name = 'Kiambu Branch';
  branch.contact = {
    _id: 'fixture:user:user2',
    name: 'Manager Ann',
    phone: '+123456789'
  };
 healthCenter.contact = {
    _id: 'fixture:user:user1',
    name: name,
    phone: '+12068881234'
  };
```